### DISCUSSION TITLE
Use of deepstack object to detect vehicles

### DISCUSSION THREAD

**Discussion by robmarkcole** [robmarkcole/HASS-plate-recognizer#26] [Ideas]
@healeydave

**Comment by healeydave** [comment]
Thanks for the tip @robmarkcole 

Unfortunately, my HA server is not suitable for the specs required for deepstack. 
I'm only running a RPi 4 with 4GB ram.

**Reply by robmarkcole** [reply]
I recommend investing in a Jetson nano, only 50 quid

**Reply by healeydave** [reply]
Interesting, is the 4GB Development kit the best one to go for?
Is it suitable for HA and presumably a lot more powerful?

**Reply by robmarkcole** [reply]
4GB is good bet. I run HA on separate machine, but nothing to stop you

**Reply by christhehoff** [reply]
FWIW - I have good success on the 2GB Nano (if you were trying to do Face and Object, I think it would struggle ....)
